yapapi/executor/ctx.py

```diff
@@ -57,8 +60,14 @@ def register(self, commands: CommandContainer):
 
 
 class _Start(Work):
+    def __init__(self, *args: Iterable[str]):
```

Comment:
```suggestion
    def __init__(self, *args: str):
```
https://www.python.org/dev/peps/pep-0484/#arbitrary-argument-lists-and-default-argument-values